src/pspm_check_model.m

```diff
@@ -2,62 +2,95 @@
-%   │             and (optional) .durations and .pmod  OR
-%   │             a cell array of struct OR
-%   │             a struct with fields 'markervalues' and 'names' (when model.timeunits
-%   │             is set to be 'markervalues')
-%   │             OR a cell array of struct
-%   ├──.timing (SF) OR .timeunits == 'whole' (SF)
-%   │             can be one of the following
-%   │                 - an SPM style onset file with two event types: onset &
-%   │                   offset (names are ignored)
-%   │                 - a .mat file with a variable 'epochs', see below
-%   │                 - a two-column text file with on/offsets
-%   │                 - e x 2 array of epoch on- and offsets, with
-%   │             e: number of epochs
-%   │             or cell array of any of these, for multiple files
-%   ├─.timeunits (GLM, SF):
-%   │             one of 'seconds', 'samples', 'markers', 'markervalues'
+%   ├─.timeunits:
+%   │ ├─.timeunits (GLM)
+%   │ │           Acceptable values:
+%   │ │             'seconds', 'samples', 'markers', or 'markervalues'
+%   │ └─.timeunits (SF)
+%   │             Acceptable values:
+%   │               'seconds', 'samples', or 'markers'
+%   ├─.timing:
+%   │ ├─.timing (DCM):
+%   │ │           Acceptable values (any of the following):
+%   │ │             * A file name/cell array of events (single session);
+%   │ │             * A cell array of file names/cell arrays.
+%   │ │           Descriptions:
+%   │ │             * When specifying file names, each file must be a *.mat file that
+%   │ │               contain a cell variable called 'events'.
+%   │ │             * Each cell should contain either one column (fixed response) or
+%   │ │               two columns (flexible response).
+%   │ │             * All matrices in the array need to have the same number of rows,
+%   │ │               i.e. the event structure must be the same for every trial. If this
+%   │ │               is not the case, include `dummy` events with negative onsets.
+%   │ ├─.timing (GLM):
+%   │ │           Acceptable values (any of the following):
+%   │ │             * A multiple condition file name (single session);
+%   │ │             * A cell array of multiple condition file names;
+%   │ │             * A struct (single session) or a cell array of struct (multiple
+%   │ │               sessions), where each struct show have the following fields:
+%   │ │               * .names (mandatory)
+%   │ │               * .onsets (mandatory)
+%   │ │               * .durations (optional)
+%   │ │               * .pmod (optional)
+%   │ │             * A struct (single session) or a cell array of struct (multiple
+%   │ │               sessions), if model.timeunits is set as 'markervalues', where each
+%   │ │               struct show have the following fields:
+%   │ │               * .markervalues
+%   │ │               * .names
+%   │ └─.timing (SF) OR .timeunits == 'whole' (SF)
```

Comment:
May I ask what `or .timeunits=='whole' (SF)` here means? Do you mean this specification is applicable if `.timeunits=='whole'`?

Reply:
If timeunits equals 'whole' then timing is not required and is in fact ignored.

Reply:
I think in this case, the definition of `.timeunits` can be completed by adding `whole` there.. Now it has only `'seconds', 'samples', 'markers', or 'markervalues'`. I have modified line 26 -- 28 and line 70 -- 71, if my understanding is correct.

Reply:
This is wrong now.

"whole" is only permitted for "SF".

Also, "DCM" needs no "timeunits" (it only accepts "timing" in seconds).

Reply:
Apologies for messing this up.. I have reverted to your latest version now...